src/super_gradients/training/pretrained_models.py

```diff
@@ -59,6 +59,9 @@
     "yolo_nas_pose_s_coco_pose": "https://sghub.deci.ai/models/yolo_nas_pose_s_coco_pose.pth",
     "yolo_nas_pose_m_coco_pose": "https://sghub.deci.ai/models/yolo_nas_pose_m_coco_pose.pth",
     "yolo_nas_pose_l_coco_pose": "https://sghub.deci.ai/models/yolo_nas_pose_l_coco_pose.pth",
+    "yolo_nas_r_s_dota2": "https://sghub.deci.ai/models/yolo_nas_r_s_dota2.pth",
```

Comment:
Can we also add a unit test verifying mAP for the pretrained checkpoints ?

Reply:
It would be hard to do in terms of getting DOTA dataset to cloud instances is not straightforward - there are no "ready-to-use" download links to automate dataset deployment process.
We don't have the dataset on tzags in standard place either :/